.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+# CONTRIBUTING
```

Comment:
For reference, see https://help.github.com/articles/setting-guidelines-for-repository-contributors/.